packages/widgets/src/weather/icon.tsx

```diff
@@ -34,6 +34,7 @@ export const WeatherIcon = ({ code, size = 50 }: WeatherIconProps) => {
 
 interface WeatherDescriptionProps {
   weatherOnly?: boolean;
+  dateFormat?: string;
```

Comment:
It should be possible to infer the type here, no?
It should be more concrete than string if there is only a certain list of available options